README.md

```diff
@@ -39,6 +39,19 @@ An Airflow container can be built with
 make build
 ```
 
+### Export Credentials
+
+For now, DAGs that use the Databricks operator won't parse until the following environment variables are set (see issue #501):
+```
+AWS_SECRET_ACCESS_KEY
+AWS_ACCESS_KEY_ID
+DB_TOKEN
+```
+
+### Migrate Database
```

Comment:
This also agrees with my observations from https://github.com/mozilla/telemetry-airflow/issues/502, and is probably enough to close it.